app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: None) -> None:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()

```

Comment:
Consider providing a descriptive error message when raising `NotVaccinatedError` to make it clear why the error is being raised.